### ISSUE TITLE
[Feature] Tag a transaction

### ISSUE THREAD

**Issue by grablair** [hammem/monarchmoney#26]
Adds the ability to add a tag to a transaction

**Comment by grablair** [comment]
Depends on #23 

That should be completed first, which will actually allow for editing a transaction. This issue isn't strictly necessary, but could be nice to have. The method created would just call the `edit_transaction` method

**Comment by hammem** [comment]
Thanks for creating these transaction issues!  This issue may also want to cover how to handle transaction IDs. 

Would some convenience methods or other baseline API be desired to make keeping track of different transactions under manipulation be helpful?

**Comment by grablair** [comment]
Makes sense to me @hammem! How about you create a [discussion](https://github.com/hammem/monarchmoney/discussions) with your ideas, and we can continue the dialogue there?
